CorePayments/src/main/java/com/paypal/android/corepayments/TrackingEventsAPI.kt

```diff
@@ -71,6 +72,7 @@ internal class TrackingEventsAPI constructor(
         const val KEY_IS_SIMULATOR = "is_simulator"
         const val KEY_MERCHANT_APP_VERSION = "mapv"
         const val KEY_ORDER_ID = "order_id"
+        const val KEY_BUTTON_TYPE = "button_type"
```

Comment:
We should update [TrackingEventsAPIUnitTest.kt](https://github.com/paypal/paypal-android/blob/main/CorePayments/src/test/java/com/paypal/android/corepayments/TrackingEventsAPIUnitTest.kt) to verify that this field is sent.